File: Python/PGS/LEVEL1/x만큼 간격이 있는 n개의 숫자/gunwoo.java
class Solution {
    public long[] solution(long x, int n) {
        long[] answer = new long[n];
        long first = 0;
        first = x;
        for(int i = 0; i < n; i++){
            answer[i] = x;
            x += first;
        }
        return answer;
    }
}

File: Algorithm/PGS/LEVEL0/5명씩/gunwoo.py
def solution(names):
    answer = []
    if len(names) < 5:
            answer.append(names[0])
    else:
        a = int(len(names)/5+1)
        for i in range(0, len(names), 5):
            answer.append(names[i])
    return answer